lang/python/python-six/Makefile

```diff
@@ -35,7 +34,7 @@ define Package/python-six/Default
   SECTION:=lang
   CATEGORY:=Languages
   SUBMENU:=Python
-  URL:=https://pypi.python.org/pypi/six
+  URL:=https://github.com/benjaminp/six
```

Comment:
I mean this URL should be unchanged.
I was replying from my phone the first time, and the comment didn't seem to go on this line

Reply:
It seems they have decided to set their GitHub page as their homepage (benjaminp/six#247), so I'm ok with this change.